dashboard/test/controllers/transfers_controller_test.rb

```diff
@@ -0,0 +1,95 @@
+require 'test_helper'
+
+class TransfersControllerTest < ActionController::TestCase
+  include Devise::TestHelpers
+
+  setup do
+    @teacher = create(:teacher)
+    sign_in(@teacher)
+
+    @word_section = create(:section, user: @teacher, login_type: 'word')
+    @word_user_1 = create(:follower, section: @word_section).student_user
+
+    @picture_section = create(:section, user: @teacher, login_type: 'picture')
+    @picture_user_1 = create(:follower, section: @word_section).student_user
+  end
+
+  test "returns an error when student ids are not provided" do
+    post :create, section_id: @picture_section.id
+    assert_response 400
+  end
+
+  test "returns an error when the section id is invalid" do
+    section_id = -100
+    post :create, section_id: section_id, student_ids: "#{@word_user_1.id.to_s}"
```

Comment:
Lint/StringConversionInInterpolation: Redundant use of `Object#to_s` in interpolation.